config/routes.rb

```diff
@@ -48,6 +48,8 @@
     end
   end
   resources :notes do
+    post :create
+    put :update
```

Comment:
I think these routes are already handled automatically by the `resources` method, according to https://guides.rubyonrails.org/routing.html#crud-verbs-and-actions. In fact, I think you're actually adding different routes compared to the Rails builtin ones, since this will create the `create` route at `/notes/create`, whereas the builtin is just at `/notes` (with a `POST` method), and yours will create the `update` route at `/notes/update`, whereas the builtin is `/notes/:id/`, where `:id` is the ID of the resource in the URL itself rather than as a parameter.